samples/advanced/data-labelling.html

```diff
@@ -3,130 +3,130 @@
-                    randomScalingFactor(),
-                    randomScalingFactor(),
-                    randomScalingFactor(),
-                    randomScalingFactor(),
-                    randomScalingFactor(),
-                    randomScalingFactor()
-                ]
-            }, {
-                type: 'bar',
-                label: 'Dataset 3',
-                backgroundColor: color(window.chartColors.green).alpha(0.2).rgbString(),
-                borderColor: window.chartColors.green,
-                data: [
-                    randomScalingFactor(),
-                    randomScalingFactor(),
-                    randomScalingFactor(),
-                    randomScalingFactor(),
-                    randomScalingFactor(),
-                    randomScalingFactor(),
-                    randomScalingFactor()
-                ]
-            }]
-        };
+		<div style="width: 75%">
+				<canvas id="canvas"></canvas>
+		</div>
+		<button id="randomizeData">Randomize Data</button>
+		<script>
+				var color = Chart.helpers.color;
+				var barChartData = {
+					labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
+					datasets: [{
+						type: 'bar',
+						label: 'Dataset 1',
+						backgroundColor: color(window.chartColors.red).alpha(0.2).rgbString(),
+						borderColor: window.chartColors.red,
+						data: [
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor()
+						]
+					}, {
+						type: 'line',
+						label: 'Dataset 2',
+						backgroundColor: color(window.chartColors.blue).alpha(0.2).rgbString(),
+						borderColor: window.chartColors.blue,
+						data: [
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor()
+						]
+					}, {
+						type: 'bar',
+						label: 'Dataset 3',
+						backgroundColor: color(window.chartColors.green).alpha(0.2).rgbString(),
+						borderColor: window.chartColors.green,
+						data: [
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor(),
+							randomScalingFactor()
+						]
+					}]
+				};
 
-        // Define a plugin to provide data labels
-        Chart.plugins.register({
-            afterDatasetsDraw: function(chart, easing) {
-                // To only draw at the end of animation, check for easing === 1
-                var ctx = chart.ctx;
+				// Define a plugin to provide data labels
+				Chart.plugins.register({
+					afterDatasetsDraw: function(chart/* , easing */) { // TODO : remove easing ?
+						// To only draw at the end of animation, check for easing === 1
```

Comment:
comment and code would be out of sync

Reply:
I would remove both the variable and the comment